src/electionguard/key_ceremony.py

```diff
@@ -163,6 +167,24 @@ class ElectionPartialKeyChallenge:
     coefficient_proofs: List[SchnorrProof]
 
 
+_T = TypeVar("_T", bound="CoordinateData")
+CONTEST_DATA_SIZE: PaddedDataSize = PaddedDataSize.Bytes_512
+
+
+@dataclass
+class CoordinateData:
```

Comment:
This class seems a bit overkill since it holds a single value, but as discussed with @keithrfung this technique simplifies padding and byte serialization/deserialization.

Reply:
Not super in love with it, but sometimes simple is better.